MediaBrowser.Tests/ConsistencyTests/Resources/StringCheckSample.xml

```diff
@@ -86,21 +84,6 @@
   <Dictionary Token="LabelChapterImageExtractionForMoviesHelp" Text="Extracting chapter images will allow clients to display graphical scene selection menus. The process can be slow, cpu-intensive and may require several gigabytes of space. It runs as a nightly scheduled task, although this is configurable in the scheduled tasks area. It is not recommended to run this task during peak usage hours." />
   <Dictionary Token="LabelEnableAutomaticPortMapping" Text="Enable automatic port mapping" />
   <Dictionary Token="LabelEnableAutomaticPortMappingHelp" Text="UPnP allows automated router configuration for easy remote access. This may not work with some router models." />
-  <Dictionary Token="HeaderTermsOfService" Text="Emby Terms of Service">
```

Comment:
I fear that Jellyfin should also have ToS and other policies... at least a notification about licensing.
But I'm no attorney, so I know very little about legal stuff.

Reply:
We can just link the GPL, their ToS was because of their hosted services which we're stripping out anyways.

Reply:
Don't we need a privacy policy, at least the one that states something like "Jellyfin won't collect any of your data to be sent anywhere"?